Gems/Atom/RHI/Code/Source/RHI/ValidationLayer.cpp

```diff
@@ -25,28 +25,43 @@ namespace AZ
             "Use this cvar to override device validation for debug builds.");
 
         static const char* ValidationCommandLineOption = "rhi-device-validation";
+        static const char* ValidationSetting = "/O3DE/Atom/rhi-device-validation";
 
-        AZStd::optional<ValidationMode> ReadValidationModeFromCommandArgs()
+        ValidationMode ReadValidationMode()
         {
+#if defined(AZ_RELEASE_BUILD)
+            // Always disabled in Release configuration.
+            return ValidationMode::Disabled;
+#else
             ValidationMode mode = ValidationMode::Disabled;
-            bool debugBuildDeviceValidationOverride = true;
-            if (auto* console = Interface<IConsole>::Get())
-            {
-                console->GetCvarValue("r_debugBuildDeviceValidationOverride", debugBuildDeviceValidationOverride);
-            }
 
-            bool isDebugBuild = AZ::RHI::BuildOptions::IsDebugBuild;
-            if (isDebugBuild && debugBuildDeviceValidationOverride)
+            // Always enabled in Debug configuration by default, unless overriden by cvar, command line or setting registry.
+            if constexpr (AZ::RHI::BuildOptions::IsDebugBuild)
             {
-                mode = ValidationMode::Enabled;
+                mode = r_debugBuildDeviceValidationOverride ? ValidationMode::Enabled : ValidationMode::Disabled;
```

Comment:
Do we not have to do 
console->GetCvarValue("r_debugBuildDeviceValidationOverride", debugBuildDeviceValidationOverride);?

Reply:
Not necessary, you can access the value directly using `r_debugBuildDeviceValidationOverride`

Reply:
Yea I believe that CVars retrieve the values by overloading the cast operator to the underlying value

Reply:
We probably could just remove the cvar completely here.